settings.py

```diff
@@ -54,10 +54,15 @@
 # will be the path to the `stylus` and `lessc` executables.
 STYLUS_BIN = os.getenv('STYLUS_BIN', path('node_modules/stylus/bin/stylus'))
 LESS_BIN = os.getenv('LESS_BIN', path('node_modules/less/bin/lessc'))
-CLEANCSS_BIN = os.getenv('CLEANCSS_BIN',
-                         path('node_modules/clean-css/bin/cleancss'))
-UGLIFY_BIN = os.getenv('UGLIFY_BIN',
-                       path('node_modules/uglify-js/bin/uglifyjs'))
+CLEANCSS_BIN = os.getenv(
+    'CLEANCSS_BIN',
+    path('node_modules/clean-css/bin/cleancss'))
```

Comment:
In my mind:

```
CLEANCSS_BIN = os.getenv(
    'CLEANCSS_BIN',
    path('node_modules/clean-css/bin/cleancss'))
```
Requires no work if you change CLEANCSS_BIN. If you change CLEANCSS_BIN:
```
OMG_WHY_DO_YOU_HATE_ME_PEP8 = os.getenv('CLEANCSS_BIN',
                         path('node_modules/clean-css/bin/cleancss'))
```
Welcome to a whole pile of code that now fails pep8 and more work.

I hate work.